force-app/main/default/aura/modalWindows/expressionsFooter/expressionsFooter.cmp

```diff
@@ -0,0 +1,9 @@
+<aura:component>
```

Comment:
if the thinking in breaking this out is that it's reusable chrome, then let's remove the "expressions" language and call it something like dialogChrome or dialogFrame or dialogButtons.